.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
```

Comment:
please add an empty line at the end of the file, to remove this EOF warning:
![image](https://user-images.githubusercontent.com/10209180/174103239-d654e77e-1496-4287-9bec-0f2dbcc91f24.png)
 
You can configure VS Code to automatically add the EOF:
https://stackoverflow.com/a/44704969